src/backend/termwiz.rs

```diff
@@ -249,12 +249,73 @@ impl Backend for TermwizBackend {
     }
 }
 
+impl From<CellAttributes> for Style {
+    fn from(value: CellAttributes) -> Self {
+        let mut style = Style::new()
+            .add_modifier(value.intensity().into())
+            .add_modifier(value.underline().into())
+            .add_modifier(value.blink().into());
+
+        if value.italic() {
+            style.add_modifier |= Modifier::ITALIC;
+        }
+        if value.reverse() {
+            style.add_modifier |= Modifier::REVERSED;
+        }
+        if value.strikethrough() {
+            style.add_modifier |= Modifier::CROSSED_OUT;
+        }
+        if value.invisible() {
+            style.add_modifier |= Modifier::HIDDEN;
+        }
+
+        style.fg = Some(value.foreground().into());
+        style.bg = Some(value.background().into());
+        #[cfg(feature = "underline_color")]
+        {
+            style.underline_color = Some(value.underline_color().into());
+        }
+
+        style
+    }
+}
+
+impl From<Intensity> for Modifier {
+    fn from(value: Intensity) -> Self {
+        match value {
+            Intensity::Normal => Modifier::empty(),
+            Intensity::Bold => Modifier::BOLD,
+            Intensity::Half => Modifier::DIM,
+        }
+    }
+}
+
+impl From<Underline> for Modifier {
+    fn from(value: Underline) -> Self {
+        match value {
+            Underline::None => Modifier::empty(),
+            _ => Modifier::UNDERLINED,
+        }
+    }
+}
+
+impl From<Blink> for Modifier {
+    fn from(value: Blink) -> Self {
+        match value {
+            Blink::None => Modifier::empty(),
+            Blink::Slow => Modifier::SLOW_BLINK,
+            Blink::Rapid => Modifier::RAPID_BLINK,
+        }
+    }
+}
+
 impl From<Color> for ColorAttribute {
     fn from(color: Color) -> ColorAttribute {
         match color {
             Color::Reset => ColorAttribute::Default,
             Color::Black => AnsiColor::Black.into(),
-            Color::Gray | Color::DarkGray => AnsiColor::Grey.into(),
+            Color::DarkGray => AnsiColor::Grey.into(),
+            Color::Gray => AnsiColor::Silver.into(),
```

Comment:
| ANSI | ratatui | termwiz |
|--------|--------|--------|
| white (37) | Gray <sup>1</sup> | Silver <sup>2</sup> |
| bright black (90) | DarkGray <sup>3</sup> | Grey | 

1: As stated in our doc "_this is sometimes called `silver` or `white` but we use `white` for bright white_"
2: As stated in [termwiz doc](https://docs.rs/termwiz/latest/termwiz/color/enum.AnsiColor.html#variant.Silver), silver is "dark white". "dark" in termwiz actually means normal in ANSI

3: As stated in our doc "_this is sometimes called `light black` or `bright black` but we use `dark gray`_"
4: According to [termwiz doc](https://docs.rs/termwiz/latest/termwiz/color/enum.AnsiColor.html#variant.Grey), grey is "bright black"

Reply:
This would probably be a breaking change, or worth noting in the changelog specifically.

Reply:
It *is* a breaking changing, but unlikely to hit anyone in a breaking way. Definitely worth mentionning in the changelog though 👍